### PR TITLE
Centralized Storage for all managers.

### PR DESCRIPTION
**Pull request by junnbang**
-can save and load all managers.
-delete your data folder before running program. Otherwise it will start with a empty storage.
-keeping addressbook for now since without it, everything breaks

P.S. tell me if any case doesn't work

### CODE REVIEW

File: src/main/java/seedu/address/model/util/SampleDataUtil.java [hunk 1]
```diff
@@ -1,45 +1,129 @@
...
 import seedu.address.model.tag.Tag;
+import seedu.address.model.task.Task;
+import seedu.address.model.task.TaskManager;
+import seedu.address.model.task.TaskStatus;
+import seedu.address.storage.CentralManager;
 
 /**
  * Contains utility methods for populating {@code AddressBook} with sample data.
  */
 public class SampleDataUtil {
```
Comment (tysng) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java]: Thanks for taking the time to update this! Much needed PR!

Comment (tysng) [reply src/main/java/seedu/address/model/util/SampleDataUtil.java]: Oh Hmm.. The magic numbers though.. If the `IdManager` had the public method to `getId` and increment counter at the same time then there will not be any magic numbers.. But I cannot think of a better way to do this given the current impl.. Let's just leave it here

File: src/main/java/seedu/address/storage/CentralManager.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import java.util.Objects;
+
+import seedu.address.model.CustomerManager;
+import seedu.address.model.DriverManager;
+import seedu.address.model.task.TaskManager;
+
+/**
+ * A wrapper manager that consists of all the managers used for saving purpose.
+ */
+public class CentralManager {
```
Comment (tysng) [inline src/main/java/seedu/address/storage/CentralManager.java]: Do we store `IdManager` in this `CentralManager` too?

**Review by tysg — DISMISSED** [review]

File: src/main/java/seedu/address/storage/CentralManager.java (hunk 2 above)
Comment (junnbang) [reply src/main/java/seedu/address/storage/CentralManager.java]: We should, ill make the changes tonight

Comment (junnbang) [reply src/main/java/seedu/address/storage/CentralManager.java]: solved

File: src/main/java/seedu/address/model/util/SampleDataUtil.java (hunk 1 above)
Comment (junnbang) [reply src/main/java/seedu/address/model/util/SampleDataUtil.java]: abstracted to static variables

**Review by tysg — APPROVED** [review]
LGTM

**Pull request merged**